blockchain/indexers/manager.go

```diff
@@ -271,10 +271,7 @@ func (m *Manager) Init(chain *blockchain.BlockChain) error {
 		err := m.db.View(func(dbTx database.Tx) error {
 			idxKey := indexer.Key()
 			hash, height, err = dbFetchIndexerTip(dbTx, idxKey)
-			if err != nil {
-				return err
-			}
-			return nil
+			return err
```

Comment:
I've mentioned this simplification before (and I like it, and use it a lot in my own code) but when asking about this as part of the btcd codebase a while ago, I was told this is not preferable since it "hides" code coverage reporting of when this function is successful vs errored.

I'm all for this change (the code behaves identically), just pointing out the inconsistency.